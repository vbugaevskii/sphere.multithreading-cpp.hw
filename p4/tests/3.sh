true&&echo 1-true 
false&&echo 1-false
true || echo 2-true
false   ||echo 2-false

true&&false&&echo 3-1
true||false&&echo 3-2

echo 123 && echo 456   &&echo 789

echo 123 > file.file && cat < file.file > file1.file && cat file1.file file1.file
