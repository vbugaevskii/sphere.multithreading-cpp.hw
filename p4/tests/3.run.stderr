Process 13933 exited: 0
Process 13934 exited: 0
Process 13935 exited: 1
Process 13936 exited: 0
Process 13937 exited: 1
Process 13938 exited: 0
Process 13939 exited: 0
Process 13940 exited: 1
Process 13941 exited: 0
Process 13942 exited: 0
Process 13943 exited: 0
Process 13944 exited: 0
Process 13945 exited: 0
Process 13946 exited: 0
Process 13947 exited: 0
Process 13948 exited: 0
